corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -835,34 +811,44 @@ def _get_relevant_cases(self, case_id):
         """
         relevant = IndexTree.get_all_dependencies(
             case_id,
-            closed_cases=self.closed_cases,
             child_index_tree=self.index_tree,
             extension_index_tree=self.extension_index_tree,
             cached_child_map=_reverse_index_map(self.index_tree.indices),
             cached_extension_map=_reverse_index_map(self.extension_index_tree.indices),
         )
-        logger.debug("Relevant cases: {}".format(relevant))
-
+        _get_logger().debug("Relevant cases of {}: {}".format(case_id, relevant))
         return relevant
 
     def _get_available_cases(self, relevant):
         """
         Mark all relevant cases that are open and have no outgoing extension indexes
-        as available. Traverse incoming extension indexes which don't lead to closed
+        as available.
+
+        Traverse incoming extension indexes which don't lead to closed
         cases, mark all touched cases as available
         """
+        def _case_is_available(case):
+            return (
+                case not in self.dependent_case_ids_on_phone  # all live cases are available
```

Comment:
this seemed logical to me, though not sure i'm overlooking something funky with extensions